assembly/common/src/main/resources/bin/sqlshell.sh

```diff
@@ -153,6 +153,8 @@ if [[ "$SPLICE_LIB_DIR" == *"##"* ]]; then
    # ends in bin, go up and over to lib
    if [[ "$CURDIR" = *"bin" ]]; then
       SPLICE_LIB_DIR="${CURDIR}/../lib"
+   elif [[ "$CURDIR" = "/var/lib/splicemachine" ]]; then #hdp2.6.3 rpm needed
```

Comment:
The whole point of this section is to handle the *fallback* behavior  if the ##SPLICE_LIB_DIR## is *not* token replaced. the default action for any built artifact should be to replace that in a platform independent way.

I dont think this is the right answer. We should find a way to replace the token for hdp2.6.3 with the indicated directory at build or install time.